avaframe/ana1Tests/energyLineTest.py

```diff
@@ -209,18 +198,19 @@ def generateCom1DFAEnergyPlot(avalancheDir, energyLineTestCfg, avaProfileMass, d
     ax3.set_yticks([avaProfileMass['z'][-1]-z0, zIntersection-z0])
     ax3.xaxis.set_major_formatter(FormatStrFormatter('%.2f'))
     ax3.yaxis.set_major_formatter(FormatStrFormatter('%.2f'))
-    text = ('Runout s diff : %.2f m \nRunout z diff : %.2f m \nRunout angle diff : %.4f° \nvelocity height rmse : %.2f m \n(energy line - alpha line)' %
+    text = ('Runout s diff : %.8f m \nRunout z diff : %.8f m \nRunout angle diff : %.8f° \nvelocity height rmse : %.8f m \n(energy line - alpha line)' %
             (runOutSError, runOutZError, runOutAngleError, rmseVelocityElevation))
     text_box = AnchoredText(text, frameon=False, loc=1, pad=0.5,
                             prop=dict(fontsize=pU.fs))
+    log.info(text)
```

Comment:
I think this should be debug level

Reply:
Isn't it nice to also see the results in the terminal?